### ISSUE TITLE
Allow splitting (active) interval by interval

### ISSUE THREAD

**Issue by xerus2000** [GothenburgBitFactory/timewarrior#385]
So, if I have an interval running and want to switch to another task or stop, I know to use `timew start <tag>` or `timew stop`, respectively.
However, since there can only be one annotation per interval, I wanted to split my active interval without changing anything. I know I can use `timew stop && timew continue`, but something like

    timew split now

would be really elegant.

**Comment by lauft** [comment]
Instead of introducing another command I would add a hint to the start command to override the current behaviour where a `start` with the same tag set is ignored:
```
 $ timew start FOO
Note: 'FOO' is a new tag.
Tracking FOO
  Started 2020-09-09T07:57:03
  Current                  03
  Total               0:00:00
$ timew start FOO # call start with the same tag set is ignored
Tracking FOO
  Started 2020-09-09T07:57:04
  Current                  04
  Total               0:00:00
$ timew start FOO :<tbd> #call start with hint
Recorded FOO
  Started 2020-09-09T07:57:03
  Ended                    28
  Total               0:00:25
Tracking FOO
  Started 2020-09-09T07:57:28
  Current                  28
  Total               0:00:00
```
The `:adjust` hint might be a candidate, on the other hand it is probably better to reserve this hint for overlap resolution... 🤔

**Comment by xerus2000** [comment]
but this won't be a new command, the split command already exists! And I think splitting by time would be a nice thing in general.

**Comment by lauft** [comment]
Splitting by time is already on the agenda, see #257. 

>... I have an interval running and want to switch to another task or stop... since there can only be one annotation per interval, I wanted to split my active interval without changing anything. ...

Reading those lines I thought you wanted to create a _new interval with the same tag set, but without the current annotation_. With `split` on the other hand the resulting intervals would all have the annotation of the previous one. This is why I proposed a variant of `start`.

**Comment by xerus2000** [comment]
Ah yes, that's fair.

An alternative for my use-case would be if you could add multiple annotations (#374) and each gets a timestamp. Basically, I want to document what I have done where.
Though I am considering learning org mode now and potentially employing that, since timewarrior seems to be mainly built for logging _when_ you work rather than _what_ you do.

**Comment by lauft** [comment]
> ... timewarrior seems to be mainly built for logging _when_ you work rather than _what_ you do

That is right. Timewarrior is a time tracking tool. For planning you can use Taskwarrior or something else. 🙂

**Comment by xerus2000** [comment]
No, I don't mean planning, I mean logging, taskwarrior doesn't really fulfill that. But I guess some notes in my favorite editor should suffice.

**Comment by sruffell** [comment]
I use our billing codes for my tags, and for the general "overhead" code, I use the annotations in timewarrior.

Like you suggested, I use `timew stop && timew continue && timew annotate 'This is the new thing'` in order to create a new record with the same tags but different annotation.

I'm looking forward to when the annotations can be specified on the command line, so that a start command with a new annotation will create a new interval.

For more detailed logging I use `jrnl`: https://jrnl.sh/

**Comment by xerus2000** [comment]
@sruffell thanks for that pointer, I have sort of already started coding a simpler version of this, but that seems much nicer!
However, I don't necessarily want all work-logs I am adding when trying to figure out a problem in my journal.
